build_bpf_modules_opensource.sh

```diff
@@ -41,7 +41,7 @@ while getopts ":hb:s:m" arg; do
     s)
       SRC_DIR="${OPTARG}"
       ;;
-    h | *) # Display help.
```

Comment:
removing * allow to run this script w/ custom defines .e.g.  ./build_bpf_modules_opensource.sh -DGUE_ENCAP